app/views/embassies/_organisation.html.erb

```diff
@@ -0,0 +1,8 @@
+<li>
+  <%= render "govuk_publishing_components/components/heading", {
+    text: ERB::Util.html_escape(organisation.locality),
```

Comment:
Why do we need the escaping here? I had a quick scan through the content item and couldn't immediately see

Reply:
I'm afraid we didn't investigate why that's there. We just copied [the partial from Whitehall](https://github.com/alphagov/whitehall/blob/f3c8ace884f8d27c91a130cd83cde96bb174023a/app/views/embassies/_organisation.html.erb) as-is in the hope that it's easier for someone to see where it came from. My guess would be to ensure that we're escaping things like `&` if they're entered in the locality field. We'll investigate further to see whether it's still necessary.

Reply:
You're right, rails does this by default in ERB templates (and has since Rails 3 😊). We've removed this call to `html_escape` in [66098ce](https://github.com/alphagov/collections/pull/3227/commits/66098ce0768d0947121f55b71ab7d16c0de2c49c)